xpk.py

```diff
@@ -642,6 +656,25 @@ class SystemCharacteristics:
     'v4-4096': SystemCharacteristics(
       '8x16x16', 512,'tpu-v4-podslice', 'ct4p-hightpu-4t', 4, AcceleratorType['TPU'], 'v4-4096'
     ),
+
+    # CPU system characteristics
+    # n2-standard-32-$VMs
+    'n2-standard-32-1': SystemCharacteristics(
+      'N/A', 1,'N/A', 'n2-standard-32', 1, AcceleratorType['CPU'], 'n2-standard-32'
+    ),
+    'n2-standard-32-2': SystemCharacteristics(
+      'N/A', 2,'N/A', 'n2-standard-32', 1, AcceleratorType['CPU'], 'n2-standard-32-2'
+    ),
+    'n2-standard-32-4': SystemCharacteristics(
```

Comment:
Hi @Obliviour 
I have used device type similar to those specified in the GPUs section "n2-standard-32-4" is "machine_type-$VMs" . Does that make sense or would you have any inputs here?

Reply:
I think this makes sense. The -$VM syntax is consistent with our other usage which is great.

Let's rename -$VM to -$vCPUs to align with the syntax in cloud.google.com:

https://cloud.google.com/compute/docs/general-purpose-machines

Reply:
There are 32vCPUs in an n2-standard-32 but it is considered 1 VM, hence the annotation. 

I am using "n2-standard-32-4" to denote 4VMs of configuration n2-standard-32. Using vCPUs instead of VMs here would be wrong actually as there are 32 vCPUs per machine and there would be 4 such machines.

Would you prefer 4-n2-standard-32 to say 4 VMs of n2-standard-32?